packages/core/useStorageAsync/index.ts

```diff
@@ -14,6 +14,11 @@ export interface UseStorageAsyncOptions<T> extends Omit<UseStorageOptions<T>, 's
    * Custom data serialization
    */
   serializer?: SerializerAsync<T>
+
+  /**
+   * On first value loaded hook.
+   */
+  onLoad?: (value: T) => void
```

Comment:
If `router` and others use `onReady`, we might also think about `onReady` to be consistent. In addition, `onReady` implies to be called once.